forms/serializers/form.py

```diff
@@ -291,12 +291,18 @@ class Meta:
         ]
 
     def get_information_checks(self, obj):
-        entry_sections = obj.entry_sections.filter(identifier="hakijan-tiedot")
+        entry_sections = obj.entry_sections.filter(
+            identifier__contains="hakijan-tiedot"
+        )
         information_checks = list(dict())
         for entry_section in entry_sections:
             for information_check in entry_section.informationcheck_set.all():
                 information_checks.append(
-                    {"id": information_check.id, "name": information_check.name}
+                    {
+                        "id": information_check.id,
+                        "name": information_check.name,
+                        "entry": information_check.entry_section.identifier,
+                    }
```

Comment:
I'd say it'd be better if this just returned all of the fields as is (i.e. also include `state`, `preparer` and `comment`), as I don't think there is any other way to get all info check objects of an answer in bulk. The UI could of course take this list and then fetch every check by ID individually, but to me that feels a bit wasteful.